automation/Extension/firefox/web-ext-config.js

```diff
@@ -10,7 +10,7 @@ const defaultConfig = {
     overwriteDest: true,
   },
   run: {
-    firefox: process.env.FIREFOX_BINARY || "firefox",
+    firefox: process.env.FIREFOX_BINARY || "firefoxdeveloperedition",
```

Comment:
Why this change? From what I can tell the unbranded build is registered as "Nightly". It seems like we shouldn't need any fallback though, as we really only want to test/crawl with the firefox binary grabbed from our crawl script and throwing and error seems preferable to silently falling back to the system's firefox or dev edition binary. (If I'm misunderstanding how this is used, lmk).

Reply:
It'd be better if this can run the bundled Fx installation, but not sure how to set that up in a cross platform way. Using firefox dev edition to start the extension is at least better than the branded firefox (which doesn't run web extension experiments).

Reply:
> (If I'm misunderstanding how this is used, lmk).

`process.env.FIREFOX_BINARY` allows developers to supply their preference when running npm commands, eg, `FIREFOX_BINARY=nightly npm start` or `FIREFOX_BINARY=/Applications/Firefox.app/Contents/MacOS/firefox-bin npm start`.